package.json

```diff
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "@18f/identity-design-system": "^7.1.0",
+    "@babel/cli": "^7.22.15",
```

Comment:
I think we should make this a `devDependencies`, because `dependencies` should be limited to what's required to build the JavaScript for IdP production. It seems like we only need it in local developer environment for publishing.

https://github.com/18F/identity-idp/blob/main/docs/frontend.md#dependencies

Reply:
resolved with commit 9127964b6045e60da7153da5c8d7cc8982ee3365